docs/contributing.rst

```diff
@@ -0,0 +1 @@
+../.github/CONTRIBUTING.rst
```

Comment:
Is it permitted for CONTRIBUTING to be in RST?

https://blog.github.com/2012-09-17-contributing-guidelines/ mentions Markdown only.